controllers/past-grants/views/index.njk

```diff
@@ -45,31 +45,54 @@
 
             <form class="u-inner" id="js-past-grants" method="get" action="" v-on:submit.prevent>
 
-                <div class="grants-searchbar">
+                <div class="grants-searchbar js-only">
                     <label class="ff-label" for="search-query">{{ copy.title }}</label>
                     <div class="form-field-addon u-margin-bottom-s">
-                        <input class="form-field-addon__input js-only"
-                               disabled="disabled"
-                               type="search"
-                               id="search-query"
-                               name="q"
-                               v-model="activeQuery">
-                        <noscript>
-                            <input class="form-field-addon__input"
-                                type="search"
-                                id="search-query"
-                                name="q"
-                                value="{{ queryParams.q }}">
-                        </noscript>
-
-                        <input class="form-field-addon__action" type="submit" value="{{ copy.submit }}" @click="updateQuery" />
-
+                        <input
+                            class="form-field-addon__input "
+                            disabled="disabled"
+                            type="search"
+                            id="search-query"
+                            name="q"
+                            v-model="activeQuery"
+                        />
+                        <input
+                            class="form-field-addon__action"
+                            type="submit"
+                            value="{{ copy.submit }}"
+                            @click="updateQuery"
+                        />
                     </div>
                     <p class="ff-help u-no-margin">
                         {{ copy.helpText }}
                     </p>
                 </div>
 
+                <noscript>
```

Comment:
Axe was complaining about duplicate IDs so to make this easier to manage I'm duplicating the whole block of markup for noscript rather than trying to interleave stuff. This does mean that we can probably move the js-version of the search bar into a vue component if we want too.